assets/lambda/sqs_etl_job_trigger_lambda.py

```diff
@@ -1,8 +1,7 @@
 """Lambda function for kicking off Glue Jobs triggered from an SQS queue."""
 
-import json
-
 import boto3
+from capepy.aws.lambda_ import EtlRecord
 
 glue_client = boto3.client("glue")
```

Comment:
In a perfect world. The current `capepy` isn't as fully featured as it should be. This can either be something we do some planning and architecting before this is merged or we can do it after